Porting/bisect-runner.pl

```diff
@@ -927,6 +927,60 @@ =head2 Point in time where code started to segfault is unknown
 
 =back
 
+=head2 Interaction of debug flags caused crash on C<-DDEBUGGING> builds
+
+=over 4
+
+=item * Problem
+
+In C<-DDEBUGGING> builds, the debug flags C<Xvt> would crash a program when F<strict.pm> was loaded via C<require> or C<use>.
+
+=item * Solution
+
+Two-stage solution.  In each stage, to shorten debugging time investigator
+made use of existing set of production releases of F<perl> built with
+C<-DDEBUGGING>.
```

Comment:
For the additional explanation you asked about you could add here something like:

> The first bisect determined the point at which code was introduced to F<strict.pm> that triggered the problem. With an understanding of the trigger, the second bisect then determined at what point such a trigger started causing a crash.